### DISCUSSION TITLE
ScriptReference/Application-version

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#625] [Notes]
# ScriptReference/Application-version

https://docs.unity3d.com/ScriptReference/Application-version.html

**Comment by Saticmotion** [comment]
While this is read-only, it is possible to update the value by changing [PlayerSettings.bundleVersion](https://docs.unity3d.com/ScriptReference/PlayerSettings-bundleVersion.html).
